specs/models/ModelDescriptor.json

```diff
@@ -43,6 +43,28 @@
     },
     "definitions": {
       "$ref": "./ModelDefinitions.json"
+    },
+    "hubCentral": {
+      "type": "object",
```

Comment:
Let's add a description to this, both for hubCentral and for modeling - for the former, something like "Captures model configuration specific to the HubCentral application", and for the latter, "Captures the location of an entity model on the modeling graph in HubCentral"